fyndiq_helpers/log_config.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import logging.config
+
+import structlog
+
+
+class HealthFilter(logging.Filter):
+    def filter(self, record):
+        request = getattr(record, 'request', None)
+        return not request.endswith('/health') if request else True
```

Comment:
Logging looks really good. Only thing I would change is `request.endswith('/health')` to `'/health' in request`, because we may have requests both to `/health` and `/health/`.

Reply:
👍